.gitignore

```diff
@@ -0,0 +1 @@
+.idea
```

Comment:
why you added this file

Reply:
I just added it because it was needed for my IDEA